UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/wwwroot/css/_print.scss

```
@@ -0,0 +1,24 @@
+﻿@media print {
+    button#Print, .msi-tabs-container, .back-btn-container, .site-footer, .site-header {
+        display: none;
+    }
+
+    .selection-details {
+        overflow-y: hidden;
+        max-height: 100% !important;
+        .warning-detail pre{
+            line-height:27px;
+            font-size:18px;
+        }
+    }
+    .footer-text{
+        margin-top:0!important;
+    }
+    #rnwInfo{
+        padding-top:0!important;
+        padding-bottom:0!important;
+    }
+    h1{
+        font-size:32px;
+    }
+}
```

By adding the following I think it would improve the printed results by avoiding page breaks in the middle of a warning.
```suggestion
    .warning-detail {
        page-break-inside: avoid;
        page-break-after: auto
    }
}
```

---

As per suggestion i have made required changes.
because off page break it create white space after some warning details.